### PR TITLE
Add sqlitebrowser

### PR DESCRIPTION
**Pull request by Mailaender**
### Please confirm your submission meets all the criteria

- [X] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:** https://github.com/sqlitebrowser/sqlitebrowser/issues/2103
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [X] Any additional patches or files have been submitted to the upstream projects concerned. https://github.com/sqlitebrowser/sqlitebrowser/pull/2605/ https://github.com/sqlitebrowser/sqlitebrowser/pull/3034

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by Mailaender** [comment]
bot, build org.sqlitebrowser.sqlitebrowser

**Comment by Mailaender** [comment]
bot, build org.sqlitebrowser.sqlitebrowser

File: org.sqlitebrowser.sqlitebrowser.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+command: sqlitebrowser
+rename-icon: sqlitebrowser
+rename-desktop-file: sqlitebrowser.desktop
+rename-appdata-file: sqlitebrowser.desktop.appdata.xml
+finish-args:
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --filesystem=home
+  - --share=network
```
Comment (hfiguiere) [inline org.sqlitebrowser.sqlitebrowser.yml:14]: does it really need network access?

File: org.sqlitebrowser.sqlitebrowser.yml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+cleanup:
+  - /include
+
+modules:
+  - name: sqlite3
```
Comment (hfiguiere) [inline org.sqlitebrowser.sqlitebrowser.yml:19]: sqlite3 is in the runtime. Isn't it enough?

**Review by hfiguiere — CHANGES_REQUESTED** [review]

File: org.sqlitebrowser.sqlitebrowser.yml (hunk 1 above)
Comment (Mailaender) [reply org.sqlitebrowser.sqlitebrowser.yml:14]: Every Qt application has this line where it says network failed in stdout, but I don't know what it does.

File: org.sqlitebrowser.sqlitebrowser.yml (hunk 2 above)
Comment (Mailaender) [reply org.sqlitebrowser.sqlitebrowser.yml]: libsqlite3 is required, but for some reason it also compiles without this.

**Comment by Mailaender** [comment]
bot, build org.sqlitebrowser.sqlitebrowser

File: org.sqlitebrowser.sqlitebrowser.yml (hunk 1 above)
Comment (tinywrkb) [reply org.sqlitebrowser.sqlitebrowser.yml]: There's support for pushing to [DBHub](https://dbhub.io/).  
Open a db and look in the `Remote` tab. You might need to enable the remote feature through settings.

Comment (Mailaender) [reply org.sqlitebrowser.sqlitebrowser.yml:14]: Added it back.

Comment (hfiguiere) [reply org.sqlitebrowser.sqlitebrowser.yml:14]: ah ok.

**Comment by justinclift** [comment]
Cool. :smile:

As a general thought, should it be using the 3.12.2 release of DB Browser for SQLite, instead of the older 3.12.1 release?

**Comment by justinclift** [comment]
Ahhh, yeah.  Looks like it just has a version number typo in the appdata.patch file. :wink:

**Comment by hfiguiere** [comment]
oh I missed that in the review :-/

**Comment by justinclift** [comment]
No worries at all.  It's a weekend, and it's not super important. :smile:

**Comment by Mailaender** [comment]
bot, build org.sqlitebrowser.sqlitebrowser

File: org.sqlitebrowser.sqlitebrowser.yml [hunk 3]
```diff
@@ -0,0 +1,35 @@
+app-id: org.sqlitebrowser.sqlitebrowser
+runtime: org.kde.Platform
+runtime-version: '5.15-21.08'
+sdk: org.kde.Sdk
+command: sqlitebrowser
+rename-icon: sqlitebrowser
+rename-desktop-file: sqlitebrowser.desktop
+rename-appdata-file: sqlitebrowser.desktop.appdata.xml
+finish-args:
```
Comment (barthalion) [inline org.sqlitebrowser.sqlitebrowser.yml:9]:
```
libGL error: MESA-LOADER: failed to retrieve device information
libGL error: MESA-LOADER: failed to open amdgpu: /usr/lib/x86_64-linux-gnu/GL/default/lib/dri/amdgpu_dri.so: cannot open shared object file: No such file or directory (search paths /usr/lib/x86_64-linux-gnu/GL/default/lib/dri, suffix _dri)
libGL error: failed to load driver: amdgpu
libGL error: failed to open /dev/dri/card0: No such file or directory
libGL error: failed to load driver: radeonsi
```

Misses `--device=dri`.

**Comment by barthalion** [comment]
Hi @justinclift, once the review is finished, should we invite someone beside you to the Flathub repo?

**Comment by justinclift** [comment]
@barthalion That's probably a really good idea, as I don't personally use Flatpaks myself.  As for who the right person is... I'm not really sure. :frowning:

**Pull request closed without merging**